inflate.c

```diff
@@ -1353,7 +1353,7 @@ int32_t Z_EXPORT PREFIX(inflateCopy)(PREFIX3(stream) *dest, PREFIX3(stream) *sou
     }
     copy->next = copy->codes + (state->next - state->codes);
     if (window != NULL) {
-        ZCOPY_WINDOW(window, state->window, 1U << state->wbits);
+        ZCOPY_WINDOW(window, state->window, (size_t)1U << state->wbits);
```

Comment:
@mtl1979  can you break out this small fix into its own PR? If not I can do it for you. It isn't necessarily related to the rest of the changes.

Reply:
Can do...

Reply:
Separated as #1429.